tsconfig.json

```diff
@@ -0,0 +1,45 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "noImplicitAny": true,
+    "module": "esnext",
+    "target": "es6",
+    "jsx": "preserve",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "typeRoots": [
+      "./types", 
+      "./node_modules/@types"
+    ]
+  },
+  "exclude": [
+    "node_modules",
+    "typings",
```

Comment:
What is `typings`?

Reply:
ah, it's an old school way of defining the `typescript types`. There use to be competitor to `definitely` types used `typings` instead of `@types`. I'll remove